docs/index.rst

```diff
@@ -38,33 +38,31 @@ pip install --pre --upgrade stellarphot
 Overview
 --------
 
-Using ``stellarphot`` starts with defining a bunch of configuration settings. Some of these,
-like details about your observing location and camera, will change infrequently.
-Others, like what size apertures to use for photometry and where in each image those
-apertures should be placed, may change from night to night.
 
-There is a graphical interface for making all of the settings below. They can also be set
-programmatically in Python or by editing a file in your favorite text editor.
 
-Provide your observatory information
--------------------------------------
+Graphical interface for generating settings
+-------------------------------------------
 
-TBD
+To generate settings using a graphical interface, start Jupyter lab. In the launcher will be a section called
```

Comment:
Is it "Jupyter lab"?  The [documentation](https://docs.jupyter.org/en/latest/) suggests it is "JupyterLab"... So does [this YouTube video](https://www.youtube.com/watch?v=dQw4w9WgXcQ).

Reply:
I think you are right...."JupyterLab"